### PR TITLE
Add some of the missing core docstrings

### PR DESCRIPTION
**Pull request by namannimmo10**
#### References to other Issues or PRs

#3904

#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by namannimmo10** [comment]
If any changes are required in the docstrings, please suggest

**Comment by czgdp1807** [comment]
Which issue is this PR fixing or refer any comment where the need for this was discussed. Thanks.

File: sympy/core/add.py [hunk 1]
```diff
@@ -461,7 +461,13 @@ def as_two_terms(self):
         return self.args[0], self._new_rawargs(*self.args[1:])
 
     def as_numer_denom(self):
+        """expression -> a/b -> a, b
```
Comment (sylee957) [inline sympy/core/add.py]: What about making a sentense like `Decomposes an expression to its numerator part and its denominator part.` and take an example to showcase something.

Comment (namannimmo10) [reply sympy/core/add.py]: Okay, I'll update.

**Comment by namannimmo10** [comment]
ping @sylee957

**Pull request merged**